src/common/security/context.go

```diff
@@ -34,6 +34,8 @@ type Context interface {
 	HasWritePerm(projectIDOrName interface{}) bool
 	// HasAllPerm returns whether the user has all permissions to the project
 	HasAllPerm(projectIDOrName interface{}) bool
+	//Get current user's all project
 	GetMyProjects() ([]*models.Project, error)
+	//Get user's role in provided project
 	GetProjectRoles(projectIDOrName interface{}) []int
```

Comment:
So after the group roles are introduced this func will still return the `user role` in contrast to `combined role`?  Could you remind me why we made such decision?

Reply:
No. It should return all roles of the user.